src/olympia/editors/views.py

```diff
@@ -645,9 +640,15 @@ def version(self):
     num_pages = pager.paginator.num_pages
     count = pager.paginator.count
 
-    try:
-        flags = ViewQueue.objects.get(id=addon.id).flags
```

Comment:
This had to go: it was re-using the crazy queue query, which was bad enough, but couldn't work with the trick used to get the latest version anyway since this is the review page, the add-on can be in any state. Tests were broken.